app/routes/application.js

```diff
@@ -0,0 +1,7 @@
+import Route from '@ember/routing/route';
+
+export default class ApplicationRoute extends Route {
+    model() {
+        return this.store.findAll('best-player');
```

Comment:
I suggest using named property in models object instead of returning just model. It's hard to understand what you are passing as a model on template below

Reply:
Done